doc/img/Node_distribution.svg

```diff
@@ -0,0 +1,1087 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The illustration had to be changed, because it contained the word "sg", which now is "sr". This new SVG file is the source of the PNG that's used in the documentation.